x/delayedack/keeper/invariants.go

```diff
@@ -16,17 +17,9 @@ const (
 
 // RegisterInvariants registers the delayedack module invariants
 func (k Keeper) RegisterInvariants(ir sdk.InvariantRegistry) {
-	ir.RegisterRoute(types.ModuleName, routeFinalizedPacket, k.PacketsFinalizationCorrespondsToFinalizationHeight)
 	ir.RegisterRoute(types.ModuleName, routeRevertedPacket, k.PacketsFromRevertedHeightsAreReverted)
 }
 
-// PacketsFinalizationCorrespondsToFinalizationHeight checks that all rollapp packets stored are set to
-// finalized status for all heights up to the latest height.
-// Skip the check if the rollapp is frozen
-func (k Keeper) PacketsFinalizationCorrespondsToFinalizationHeight(ctx sdk.Context) (string, bool) {
-	return k.packetsCorrespondsToStatusHeight(checkFinalizedPackets, false)(ctx)
-}
-
```

Comment:
it might be better to keep a similar invariant which is that all packets which not in finalized height are not finalized status